paddle/fluid/framework/ir/mkldnn/cpu_quantize_pass_tester.cc

```diff
@@ -604,6 +605,63 @@ TEST(CpuQuantizePass, matmul_not_quantized) {
   MainTestMatmul(BuildProgramDescMatmulNotQuantized(), matmul_count,
                  quant_count, dequant_count, added_nodes_count, 1.0f);
 }
+
+static const std::initializer_list<std::string> variable_names_scale = {
+    "a", "b", "c", "d", "e", "f"};
+
+ProgramDesc BuildProgramDescScaleRelocate() {
+  ProgramDesc prog;
+  for (auto& v : variable_names_scale) {
+    prog.MutableBlock(0)->Var(v);
+  }
+  SetOp(&prog, "fc", "Fc", {"a"}, {"b"}, false);
+  SetOp(&prog, "reshape2", "Reshape", {"b"}, {"c"}, false);
+  SetOp(&prog, "transpose2", "Transpose", {"c"}, {"d"}, false);
+  SetOp(&prog, "scale", "Scale", {"d"}, {"e"}, false);
+  SetOp(&prog, "matmul", "Matmul", {"e"}, {"f"}, false);
```

Comment:
Probably the test can forgo the second input of matmul, but I would keep the graph as legitimate as possible and add the second input to `matmul`.